sdk/core/core-amqp/src/retry.ts

```diff
@@ -59,6 +63,12 @@ export interface RetryConfig<T> {
    * Extremely useful in providing better debug logs.
    */
   operationType: RetryOperationType;
+  /**
+   * @property {number} operationTimeoutInMs Maximum time allowed for within which an operation is
+   * expected to be completed. If operation takes longer, it will be cancelled, considered as failed
+   * with a retryable error and will be retried again on based on supplied `RetryConfig`
```

Comment:
This can be made simpler

> Maximum time in milliseconds allowed for each attempt to complete. Failure to complete will result in the retryable error `OperationTimeoutError` being thrown.